.env.example

```
@@ -11,9 +10,26 @@ DB_NAME=
 DB_SYNCHRONIZE=true
 DB_SCHEMA:
 
-#list of permissions.
-SCOPES= []
-EGO_URL: "https://ego.url"
+# ----- Auth -----
+
+# client id and secret as present is keycloak or ego for this service
+CLIENT_ID=
+CLIENT_SECRET=
```

Can we make this name more specific to our Ego auth setup? I think it could get confusing in the long config.

```suggestion
EGO_CLIENT_ID=
EGO_CLIENT_SECRET=
```

---

Both keycloak and ego need a client id and secret so this field will relate to either of the auth mechanisms employed at the time of use.

---

maybe `AUTH_CLIENT_ID` then? Its also not a problem to have separate properties for each:
```ts
EGO_CLIENT_ID=
EGO_CLIENT_SECRET=
KEYCLOAK_CLIENT_ID=
KEYCLOAK_CLIENT_SECRET=
```
Makes it very obvious when configuring what the value is used for. These would only be used when the auth strategy is set to EGO or KEYCLOAK.

---

How about having just a single set like 
`AUTH_SERVER_CLIENT_ID=
AUTH_SERVER_CLIENT_SECRET=`

That way we can avoid having extra config properties while still making it clear what it stands for.

---

yes thats better than just `client_id`.